app/main.py

```diff
@@ -1,6 +1,18 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {"q": "Paris", "key": os.environ["API_KEY"]}
```

Comment:
Use constants for params:
CITY = "Paris"
API_KEY = os.environ["API_KEY"]
URL = "http://api.weatherapi.com/v1/current.json"

Reply:
I don't use it more than one time so it has no sense

Reply:
@Danil228  is right, it is better to make them constant for better code style